packages/components/src/Markdown/Markdown.test.tsx

```diff
@@ -1,168 +1,71 @@
 import React from "react";
 import renderer from "react-test-renderer";
-import { cleanup } from "@testing-library/react";
 import { Markdown } from ".";
 
-afterEach(cleanup);
-
 it("renders Text Component", () => {
   const tree = renderer.create(<Markdown content="Paragraph" />).toJSON();
-  expect(tree).toMatchInlineSnapshot(`
-    <div
-      className="base"
-    >
-      <p
-        className="base regular base greyBlueDark"
-      >
-        Paragraph
-      </p>
-    </div>
-  `);
+  expect(tree).toMatchSnapshot();
```

Comment:
I wonder if we should update the `plop` generator to not use inline snapshot…

Reply:
Yah, I plan to update the default content too.. I'll add a task